controllers/policyserver_controller_test.go

```diff
@@ -559,6 +560,79 @@ var _ = Describe("PolicyServer controller", func() {
 			}).Should(Succeed())
 		})
 
+		It("the deployment must have the configMap version as an annotation", func() {
+			policyServer := policyServerFactory(policyServerName)
+			createPolicyServerAndWaitForItsService(policyServer)
+			Eventually(func() error {
+				configmap, err := getTestPolicyServerConfigMap(policyServerName)
+				if err != nil {
+					return err
+				}
```

Comment:
I think this could be moved right outside of the `eventually` block. The confimap is not supposed to change while this test runs, isn'it it?